### PR TITLE
RPG Style Inventory Multiplayer Compatibility

### PR DESCRIPTION
**Pull request by Thomas107500**
Tested with different apparel and a weapon, no desyncs or errors

### CODE REVIEW

File: Source/Mods/RPGStyleInventory.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+namespace Multiplayer.Compat
+{
+    /// <summary>RPG Style Inventory by Sandy</summary>
+    /// <see href="https://steamcommunity.com/sharedfiles/filedetails/?id=1561221991"/>
+    [MpCompatFor("Sandy.RPGStyleInventory")]
+    class RPGStyleInventory
+    {
+
+        private static Type type = AccessTools.TypeByName("Sandy_Detailed_RPG_Inventory.Sandy_Detailed_RPG_GearTab");
```
[the remark this answers is not in the document]
Comment (Thomas107500) [reply Source/Mods/RPGStyleInventory.cs]: Completed

**Comment by Thomas107500** [comment]
It is all thats required :D, this SetContext is awesome :D

File: Source/Mods/RPGStyleInventory.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    class RPGStyleInventory
+    {
+
+
+        public RPGStyleInventory(ModContentPack mod)
+        {
+            MP.RegisterSyncWorker<ITab_Pawn_Gear> (SyncITab, AccessTools.TypeByName("Sandy_Detailed_RPG_Inventory.Sandy_Detailed_RPG_GearTab"));
+            MP.RegisterSyncMethod(AccessTools.TypeByName("Sandy_Detailed_RPG_Inventory.Sandy_Detailed_RPG_GearTab"), "InterfaceDrop").SetContext(SyncContext.MapSelected);
+            MP.RegisterSyncMethod(AccessTools.TypeByName("Sandy_Detailed_RPG_Inventory.Sandy_Detailed_RPG_GearTab"), "InterfaceIngest").SetContext(SyncContext.MapSelected);
+
```
Comment (notfood) [inline Source/Mods/RPGStyleInventory.cs:21]: Hold on, I asked for it to be made into a local variable so it doesn't linger around.

Comment (Thomas107500) [reply Source/Mods/RPGStyleInventory.cs:21]: Ah ok i will fix it right away

Comment (Thomas107500) [reply Source/Mods/RPGStyleInventory.cs:23]: OK fixed

**Comment by Thomas107500** [comment]
oh just a tiny bit of space i deleted :P

**Comment by notfood** [comment]
Closed by #26

**Pull request closed without merging**